doc/howto/storage_pools.md

```diff
@@ -196,13 +211,33 @@ Storage pool my-pool pending on member vm03
 Storage pool my-pool created
 ```
 
-Also see {ref}`cluster-config-storage`.
+#### Create a remote storage pool
 
-```{note}
-For most storage drivers, the storage pools exist locally on each cluster member.
-That means that if you create a storage volume in a storage pool on one member, it will not be available on other cluster members.
+The following series of commands sets up a Ceph RBD storage pool with the name `my-remote-pool` and the on-disk name `my-osd` on three cluster members:
+As the configuration setting isn't member specific it has to be set when creating the actual storage pool:
```

Comment:
```suggestion
The following series of commands sets up a Ceph RBD storage pool with the name `my-remote-pool` and the on-disk name `my-osd` on three cluster members.
Because the {config:option}`storage-ceph-pool-conf:ceph.osd.pool_name` configuration setting isn't member-specific, it must be set when creating the actual storage pool:
```

Reply:
Thanks, fixed.